src/components/molecules/Chatbox/Chatbox.scss

```diff
@@ -28,6 +28,7 @@
 
   &__input {
     width: 100%;
+    padding-right: 10px;
```

Comment:
Why are we adding this padding here?

Reply:
It appears to equally pad the emoji icon next to the message field from the 'send' button. Otherwise, the emoji button appears to close to the message field.

Reply:
Looks like it's no longer the case
![Uploading image.png…]()